Sensors/SMCBatteryManager/SMCBatteryManager.cpp

```diff
@@ -61,6 +61,8 @@ bool SMCBatteryManager::start(IOService *provider) {
 	
 	DBGLOG("bmgr", "AppleSMC is available now");
 
+	BatteryManager::getShared()->start();
```

Comment:
Ugh, are you sure of this change?

Reply:
I moved it before SMC key submission so it's possible to updated configuration and values, which is used to decide whether `TB0T`s should present. The ACPI polls should work well on its own, or did I missing something?

Reply:
Right, that is probably ok, just a little worried of the organisation…